libkbfs/conflict_resolver.go

```diff
@@ -2896,77 +2896,20 @@ func (cr *ConflictResolver) syncTree(ctx context.Context, lState *lockState,
 	return bps, nil
 }
 
-// addUnrefToFinalResOp makes a resolutionOp at the end of opsList if
-// one doesn't exist yet, and then adds the given pointer as an unref
-// block to it.
-func addUnrefToFinalResOp(ops opsList, ptr BlockPointer) opsList {
-	resOp, ok := ops[len(ops)-1].(*resolutionOp)
-	if !ok {
-		resOp = newResolutionOp()
-		ops = append(ops, resOp)
-	}
-	resOp.AddUnrefBlock(ptr)
-	return ops
-}
-
-// calculateResolutionBytes figured out how many bytes are referenced
-// and unreferenced in the merged branch by this resolution.  It
-// should be called before the block changes are unembedded in md.  It
-// returns the list of blocks that can be remove from the flushing
-// queue, if any.
-func (cr *ConflictResolver) calculateResolutionUsage(ctx context.Context,
+// updateResolutionUsage figures out how many bytes are referenced and
+// unreferenced in the merged branch by this resolution.  Only needs
+// to be called for non-squash resolutions.
+func (cr *ConflictResolver) updateResolutionUsage(ctx context.Context,
```

Comment:
This whole function is just refactored out of the old `calculateResolutionUsage()` unchanged.